Doc/whatsnew/3.13.rst

```diff
@@ -280,6 +280,11 @@ Other Language Changes
   class scopes are not inlined into their parent scope. (Contributed by
   Jelle Zijlstra in :gh:`109118` and :gh:`118160`.)
 
+* Classes have a new :attr:`!__firstlineno__` attribute,
+  populated by the compiler, with the line number of the first line
+  of the class definition.
```

Comment:
If the class has decorators then I think this is the first line of the first decorator. Need a test for this case, and probably to mention in the doc.

Reply:
There are existing tests, and they are passed.

I'll add explicit mentioning of this fact, but AFAIK it was not specified for `co_firstlineno` etc.